business/suggestionService.js

```diff
@@ -80,7 +79,7 @@ class SuggestionService {
       suggestions,
       field,
       suggestionDefinitions.map(suggestion => {
-        return { value: get(suggestion, field) }
+        return { value: get(suggestion, field), version: get(suggestion, 'coordinates.revision') }
```

Comment:
We should add all the missing fields as for the sample in issue https://github.com/clearlydefined/service/issues/290 : 
- date
- curator
- curation